src/components/Toaster/__stories__/ToasterShowcase.tsx

```diff
@@ -292,6 +303,12 @@ export const ToasterDemo = ({
         </Button>
     );
 
+    const withoutTitleToastBtn = (
+        <Button view="outlined" size="l" onClick={createWithoutTitleToast} style={btnStyle}>
+            Create toast without title
+        </Button>
+    );
```

Comment:
We have option to add content for Toast, it is better to add same option for title, instead of another button